lib/screens/account/account_details_screen.dart

```diff
@@ -0,0 +1,365 @@
+import 'dart:developer';
+
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:go_router/go_router.dart';
+
+import '../../config.dart';
+import '../../router/routing_constants.dart';
+import '../../widgets/modal.dart';
+import '../../widgets/screen.dart';
+
+class AccountDetailsScreen extends StatelessWidget {
+  const AccountDetailsScreen({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    const String iban = 'DE43 1101 0100 2919 3290 34';
+    const String bic = 'SOLARIS35';
+
+    return Screen(
+      scrollPhysics: const NeverScrollableScrollPhysics(),
+      titleTextStyle: const TextStyle(
+        fontSize: 16,
+        height: 24 / 16,
+        fontWeight: FontWeight.w600,
+      ),
+      backButtonIcon: const Icon(Icons.arrow_back, size: 24),
+      customBackButtonCallback: () {
+        context.push(homeRoute.path);
+      },
+      centerTitle: true,
+      hideAppBar: false,
+      hideBackButton: false,
+      hideBottomNavbar: true,
+      child: Column(
+        children: [
+          Padding(
+            padding:
+                ClientConfig.getCustomClientUiSettings().defaultScreenPadding,
+            child: Column(
+              mainAxisAlignment: MainAxisAlignment.center,
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: [
+                const Text(
+                  'Account',
+                  style: TextStyle(
+                    fontSize: 32,
+                    height: 40 / 32,
+                    fontWeight: FontWeight.w600,
+                  ),
+                ),
+                const SizedBox(height: 24),
+                Column(
+                  mainAxisAlignment: MainAxisAlignment.center,
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  children: [
+                    const Text(
+                      'Details',
+                      style: TextStyle(
+                        fontSize: 18,
+                        height: 24 / 18,
```

Comment:
E putin ciudat sa faci calculele direct aici. Poti sa pui direct 1.3 si gata. Desi la text pe o singura linie nici nu ai nevoie de proprietatea aia deloc

Reply:
Am ales sa pun sub forma unei fractii din 2 motive:
1. un numar rational nu are conexiune cu valorile din figma si am crezut ca ar putea crea confuzie. Sub forma unei fractii cred ca e mai usor de inteles
2. permite flexiblitate in cazul modificarii layout-ului pentru 1 dintre valori: ori inaltime font, ori inaltime rand